ext/node/polyfills/tls.ts

```diff
@@ -30,7 +34,23 @@ export function getCiphers() {
   return Object.keys(cipherMap).map((name) => name.toLowerCase());
 }
 
-export const rootCertificates = undefined;
+const certs = {};
+let rootCertificates_;
+function cacheRootCertificates() {
+  rootCertificates_ = ObjectFreeze(op_get_root_certificates());
+}
+ObjectDefineProperty(certs, "rootCertificates", {
+  __proto__: null,
+  configurable: false,
+  enumerable: true,
+  get: () => {
+    // Out-of-line caching to promote inlining the getter.
+    if (!rootCertificates_) cacheRootCertificates();
+    return rootCertificates_;
+  },
+});
+
+export const { rootCertificates } = certs;
```

Comment:
This destructuring assignment invokes the getter making it non-lazy. Any ideas on how to make it lazy load?